projects/storefrontstyles/scss/themes/calydon/components/content/banner/_banner.scss

```diff
@@ -0,0 +1,6 @@
+%cx-banner-calydon {
+  &:focus-within {
```

Comment:
we shouldn't rely on `:focus-within` pseudo-class, because it is not supported in Edge